common/config/rush/pnpm-config.json

```diff
@@ -91,7 +91,8 @@
   "globalOverrides": {
     "jsonwebtoken@<=8.5.1": ">=9.0.0",
     "jsonwebtoken@<9.0.0": ">=9.0.0",
-    "xml2js@<0.5.0": ">=0.5.0"
+    "xml2js@<0.5.0": ">=0.5.0",
+    "yaml@<2.2.2": ">=2.2.2"
```

Comment:
Is this change needed?

Reply:
I had the base branch: main
That change is in develop as a fix from pnpm audit